https/gifts.js

```diff
@@ -16,140 +16,242 @@ const { EVENT } = require("../config/constants");
 const db = require("../config/db");
 
 /**
- * @namespace sendGift
- * @return {sendGift~inner} - the returned function
+ * @namespace updateGiftSentStatus
+ * @return {updateGiftSentStatus~inner} - the returned function
  */
-const sendGift = functions.https.onCall(
+const updateGiftSentStatus = functions.https.onCall(
   /**
-   * marks the gift sent on boith the gifter and giftees account
+   * Updates the sent status of the gift
    * @inner
    * @param {object} data - details about the giftee
    * @param {string} data.user - user object or uid
-   * @param {boolean} data.value - marks it either true or false
-   * @param {string} data.gifteeGameAccountUUID - UUID of the giftee, if you do not know it
-   * @param {object} [context] - This is used by firebase, no idea what it does, I think its added automatically
```

Comment:
This may be required, I believe it is used by firebase, if stuff breaks then it could be a good idea to re-add it